internal/examples/config/client.yaml

```diff
@@ -0,0 +1,17 @@
+name: myservice-client
+outbounds:
+  myservice:
+    unary:
+      tchannel:
+        address: localhost:4040
+      # http:
```

Comment:
We should create a ticket to investigate making the http inbound and outbound config a bit more symmetric (if possible).